libnavui-dropin/src/main/java/com/mapbox/navigation/dropin/binder/map/RoutePreviewMapBinder.kt

```diff
@@ -25,12 +25,14 @@ internal class RoutePreviewMapBinder(
                 value,
                 navigationViewContext.viewModel.locationViewModel,
             ),
-            reloadOnChange(navigationViewContext.options.routeLineOptions) { lineOptions ->
-                RouteLineComponent(
-                    value,
-                    lineOptions,
-                    navigationViewContext.viewModel.routesViewModel
-                )
+            reloadOnChange(navigationViewContext.loadedMapStyle) {
+                reloadOnChange(navigationViewContext.options.routeLineOptions) { lineOptions ->
```

Comment:
Maybe not in this PR, but it would be nice to be able to combine these

```suggestion
            reloadOnChange(navigationViewContext.loadedMapStyle, navigationViewContext.options.routeLineOptions) { _, lineOptions ->

```

Reply:
Yeah... We could follow coroutines flow transform approach and introduce something similar to `collect(): Flow`.

Reply:
Done.